prow/pj-on-kind.sh

```diff
@@ -13,117 +15,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# */
-
-# This script is copied from k/test-infra
-# https://github.com/kubernetes/test-infra/blob/488e767e326f6c7189cbf0682e7f926040ae959c/prow/pj-on-kind.sh
-# Runs prow/pj-on-kind.sh with config arguments specific to Jetstack Prow config.
-# Requries go, docker, and kubectl.
 
-# Example usage:
-# ./prow/pj-on-kind.sh ci-cert-manager-e2e-v1-21
+# Runs prow/pj-on-kind.sh with config arguments specific to the prow.k8s.io instance.
 
```

Comment:
I don't think we want to remove this comment or add a reference to `prow.k8s.io` which is not what we are targeting